CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-# [4.0.0](https://github.com/imgix/imgix-rb/compare/3.1.0...4.0.0) - December 03, 2019
+## [4.0.1](https://github.com/imgix/imgix-rb/compare/3.1.0...4.0.0) - June 10, 2020
```

Comment:
```suggestion
## [4.0.1](https://github.com/imgix/imgix-rb/compare/4.0.0...4.0.1) - June 10, 2020
```

Reply:
Thanks! I pushed before correcting that smh (I was thinking, "I literally did this," yeah but the diff don't lie haha)

Reply:
Actually, I guess sometimes vscode needs its directory reloaded? Clicked on the "refresh explorer" button and then my most recent changes appeared in the integrated terminal. Good to know.